hal/network/lwip/ppp_client.cpp

```diff
@@ -31,6 +31,10 @@ extern "C" {
 #include "system_error.h"
 #include "lwiplock.h"
 
+#undef LOG_COMPILE_TIME_LEVEL
+#define LOG_COMPILE_TIME_LEVEL LOG_LEVEL_ALL // include errors
```

Comment:
Why only errors? :) It will include trace, info, warning, error and critical levels.

Reply:
Some might have thought to just enable TRACE for AT commands, so it was a comment to that effect. Perhaps I’ll just remove the comments on these.

Reply:
Some might have thought to just enable TRACE for AT commands, so it was a comment to that effect. Perhaps I’ll just remove the comments on these.

Reply:
Updated and removed the comments.